index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
+
+        <!--Header-->
+        <header>
+            <h1>John Doe</h1>
+
+            <div id="navig">
+                <label for="humberger" id="hum">&#9776;</label>
+                <input type="checkbox" id="humberger"/>
+                <div id="navitems">
+                    <a href="#">Portfolio</a>
+                    <a href="#">About</a>
+                    <a href="#">Contact</a>
+                    <a href="#"><img alt="Mail" src="/assets/mail-icon.svg" width="20" height="30"></a>
+                </div>
+                </div>
+</header>
+<!--Toolbar-->
+<section id="toolbar">
+    <div id="heytext">
+        <p>Hey There.</p>
+        <p>I’m Alex</p>
+    </div>
+    <div id="whoami">
+        <p>I’am a Software Developer</p>
+        <p>I can help you build a product , feature<br>
+            or website Look through some of my<br> work and experience! If you like what you<br>
+             see and have a project you need coded,<br> 
+        don’t hestiate to contact me.</p>
+    </div>
+    
+
+</section>
+<!--Social media-->
+<section id="socials">
+    <div id="socialicons">
+        <a href="#"><img alt="git" src="assets/githubicon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="linkein" src="assets/linkedinicon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="angel" src="assets/angelisticon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="twitter" src="assets/twittericon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="medium" src="assets/mediumicon.svg" width="50" height="60"></a>
+    </div>
+    <div id="imagearea">
+
+    </div>
+</section>
```

Comment:
- [ ] The social links are part of the same headline section so there is no need of creating a different section for it. It also looks like a list so `<ul>` with `<li>` for the items would make it more semantically correct. Also, remember that the icons are links to your social pages, so remember to use a proper tag for that. This will make a better user experience.